### DISCUSSION TITLE
Is there a simple way to re-pair LR devices from mesh to LR?

### DISCUSSION THREAD

**Discussion by jasonkelly214** [zwave-js/zwave-js#7084] [Q&A]
So now that my z-wave network is back up and running again (10 months later) I am picking up where I left off in building out my network on my property. Since LR support is now available, I want to migrate a lot of my devices to use LR instead, as I was having to hop stuff 3 or 4 times to get back to my controller. Is there a simple way to migrate existing nodes to LR? For the most part removing and re-adding will be fine, but some devices are really annoying to do since they lose their configuration data when you remove them.

**Comment by kpine** [comment]
No, the only way is to exclude them and re-include them with SmartStart.

For config settings, with Z-Wave JS UI you could create some Driver function snippet to backup and restore the config settings. I'd use the built-in clone-config as inspiration.

![image](https://github.com/user-attachments/assets/683f665d-6441-4e0d-be64-669776b61b75)

**Reply by jasonkelly214** [reply]
Thanks! Looks like I have less to do than I thought. It seems zooz kept the same part numbers when they changed the zwave chip rev so looking online at the docs for the ZSE41. ZEN17, etc I thought all my devices had 800 chips, but since I bought them last year they're in fact all using 700 chips.

**Comment by AlCalzone** [comment]
The above answer is correct, just for completeness:
If you've already included the devices via SmartStart before, you can just exclude them, toggle the SmartStart entry from Z-Wave to Long Range, and enable the entry again. After re-powering the device or waiting a bit, it should include via LR again.
